components/admin_console/permission_schemes_settings/guest_permissions_tree/guest_permissions_tree.jsx

```diff
@@ -39,7 +39,11 @@ export default class GuestPermissionsTree extends React.PureComponent {
 
         this.ADDITIONAL_VALUES = {
             guest_edit_post: {
-                editTimeLimitButton: <EditPostTimeLimitButton onClick={this.openPostTimeLimitModal}/>,
+                editTimeLimitButton:
+    <EditPostTimeLimitButton
+        onClick={this.openPostTimeLimitModal}
+        isDisabled={this.props.readOnly}
+    />,
```

Comment:
is the spacing off here?

Reply:
```suggestion
                editTimeLimitButton: (
                    <EditPostTimeLimitButton
                        onClick={this.openPostTimeLimitModal}
                        isDisabled={this.props.readOnly}
                    />
                ),
```

Reply:
Not sure why ESLint is making me put the spacing in as shown.

Reply:
have you tried by encapsulating the JSX element in parentheses as ive done above?

Reply:
I pulled the branch down and 
```
        this.ADDITIONAL_VALUES = {
            guest_edit_post: {
                editTimeLimitButton: (
                    <EditPostTimeLimitButton
                        onClick={this.openPostTimeLimitModal}
                        isDisabled={this.props.readOnly}
                    />
                ),
            },
        };
``` 
passes through the linter for me